src/scenes/Summary/summaryTable.ts

```diff
@@ -159,13 +141,13 @@ function getSummaryTableQueryRunner(metrics: DataSourceRef, sm: DataSourceRef) {
               operation: 'aggregate',
             },
             'Value #state': {
-              aggregations: ['last'],
-              operation: 'aggregate',
-            },
-            'Value #uptime': {
-              aggregations: ['mean'],
+              aggregations: ['lastNotNull'],
               operation: 'aggregate',
             },
+            // 'Value #uptime': {
+            //   aggregations: ['mean'],
+            //   operation: 'aggregate',
+            // },
```

Comment:
Maybe just remove this now?

Reply:
I was hoping some solution was going to appear to allow me not to get rid of this....alas